docs/cloud-messaging/receive.md

```diff
@@ -409,3 +409,97 @@ experimentalSetDeliveryMetricsExportedToBigQueryEnabled(messaging, true);
 ```
 
 Don't forget to run `yarn build` in order to export the new version of your service worker to the `web` folder.
+
+
+## (Advanced, Optional) Allowing Notification Images on iOS
```

Comment:
Suggest just "Display images in notifications on iOS" unless you have strong reasons for using "allow."  Note just init cap for headings.